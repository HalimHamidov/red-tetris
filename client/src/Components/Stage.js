import React from "react";
import Cell from "./Cell";
import { StyleStage } from "./styling/StyledStage";
import GameOver from "./GameOver";

const Stage = ({ stage, gameOver, player }) => {
  if (!stage) {
    return <span>Loading...</span>;
  }
  return (
    <StyleStage width={stage[0].length} height={stage.length}>
      {gameOver && <GameOver {...{ player }} />}
      {stage.map((row, i) =>
        row.map((cell, x) => <Cell key={x} type={cell[0]} />)
      )}
    </StyleStage>
  );
};

export default Stage;
