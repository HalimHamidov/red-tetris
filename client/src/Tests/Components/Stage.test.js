import { render } from "@testing-library/react";
import Stage from "../../Components/Stage";

test("Test Stage", () => {
    render( <
        Stage / >
    );
});