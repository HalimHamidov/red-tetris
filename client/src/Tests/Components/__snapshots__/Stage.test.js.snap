// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Stage Gameover rendering  1`] = `
<div
  className="sc-gsDKAQ bzNBex"
  height={12}
  width={20}
>
  <div
    className="sc-dkPtRN huOrfL"
  >
    <div
      className="sc-hKwDye LoZPU"
    >
      <img
        data-testid="gameOver-Won-img"
        src="Tetris-Founder.png"
        style={
          Object {
            "width": "255px",
          }
        }
      />
      <h1>
        You Win! Congratulations! 
      </h1>
      <p>
        Wait for host to restart the game
      </p>
    </div>
  </div>
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdvvtL yBcMv"
    color="0, 0, 0"
    type={0}
  />
</div>
`;

exports[`Test Stage rendering  1`] = `
<span>
  Loading...
</span>
`;
